config/config.exs

```diff
@@ -4,3 +4,7 @@ config :geocoder, :worker_pool_config, [
   size: 4,
   max_overflow: 2
 ]
+
+config :geocoder, :worker, [
+  provider: Geocoder.Providers.GoogleMaps # OpenStreetMaps
```

Comment:
I answered in the conversation view since these are three comments playing together.